static/css/eval_retrieval_generation.scss

```diff
@@ -462,7 +477,7 @@ label {
   display: flex;
   flex-direction: column;
   width: 50%;
-  height: 100%;
+  height: 70%;
```

Comment:
can you check how this looks with the original RIG/RAG eval tools

Reply:
I think .sxs-pane only gets used in SxS tool.

Reply:
ah, yes, did not see this was under sxs